### DISCUSSION TITLE
Whats's in Experiments Directory?

### DISCUSSION THREAD

**Discussion by SHVM-09** [saadeghi/daisyui#2804] [Q&A] [answered]
The experiments directory assesses Tailwind CSS, DaisyUI, and Bootstrap, but details regarding the criteria for comparison and the resulting conclusions are not provided. Please share information about the specific criteria used in the comparison and the conclusions drawn from the analysis.

~ Awaiting Answer

**Comment by saadeghi** [comment, marked as answer]
https://github.com/saadeghi/daisyui/tree/master/src/experiments/playground is an isolated Astro setup for playing with components, used for development and debugging.  

https://github.com/saadeghi/daisyui/tree/master/src/experiments/compare/ includes a login page designed with 3 different setups, to compare the size of HTML and CSS for each setup and the number of class names.

This directory is for internal use only You can play with them but you don't need to.

Let me know if you have any questions.

**Reply by SHVM-09** [reply]
Thanks for the response! 

~Regards!
